src/main/java/net/sacredlabyrinth/phaed/simpleclans/CommandHelper.java

```diff
@@ -10,6 +10,10 @@
 import java.lang.reflect.Field;
 import java.util.Arrays;
 
+/***
+ * @deprecated No usages
+ */
+@Deprecated
 public class CommandHelper {
```

Comment:
Can be removed

Reply:
What about [DamageRecord.java](https://github.com/RoinujNosde/SimpleClans/blob/dev/src/main/java/net/sacredlabyrinth/phaed/simpleclans/DamageRecord.java)? It is deprecated too.

Reply:
Sure